macOS/DuckDuckGo/TabBar/View/TabBarViewController.swift

```diff
@@ -533,81 +501,6 @@ final class TabBarViewController: NSViewController, TabBarRemoteMessagePresentin
             }
     }
 
-    private func subscribeToPinnedTabsViewModelInputs() {
-        guard let pinnedTabsViewModel = pinnedTabsViewModel else { return }
-
-        tabCollectionViewModel.$selectionIndex
-            .map { [weak self] selectedTabIndex -> Tab? in
-                switch selectedTabIndex {
-                case .pinned(let index):
-                    return self?.pinnedTabsViewModel?.items[safe: index]
-                default:
-                    return nil
-                }
-            }
-            .assign(to: \.selectedItem, onWeaklyHeld: pinnedTabsViewModel)
-            .store(in: &cancellables)
-
-        Publishers.CombineLatest(tabCollectionViewModel.$selectionIndex, $tabMode)
-            .map { selectedTabIndex, tabMode -> Bool in
-                if case .unpinned(0) = selectedTabIndex, tabMode == .divided {
-                    return false
-                }
-                return true
-            }
-            .assign(to: \.shouldDrawLastItemSeparator, onWeaklyHeld: pinnedTabsViewModel)
-            .store(in: &cancellables)
-    }
-
-    private func subscribeToPinnedTabsViewModelOutputs() {
-        guard let pinnedTabsViewModel = pinnedTabsViewModel else { return }
-
-        pinnedTabsViewModel.tabsDidReorderPublisher
-            .sink { [weak self] tabs in
-                self?.tabCollectionViewModel.pinnedTabsManager?.tabCollection.reorderTabs(tabs)
-            }
-            .store(in: &cancellables)
-
-        pinnedTabsViewModel.$selectedItemIndex.dropFirst().removeDuplicates()
-            .compactMap { $0 }
-            .sink { [weak self] index in
-                self?.deselectTabAndSelectPinnedTab(at: index)
-            }
-            .store(in: &cancellables)
-
-        pinnedTabsViewModel.$hoveredItemIndex.dropFirst().removeDuplicates()
-            .debounce(for: 0.05, scheduler: DispatchQueue.main)
-            .sink { [weak self] index in
-                self?.pinnedTabsViewDidUpdateHoveredItem(to: index)
-            }
-            .store(in: &cancellables)
-
-        pinnedTabsViewModel.contextMenuActionPublisher
-            .sink { [weak self] action in
-                self?.handlePinnedTabContextMenuAction(action)
-            }
-            .store(in: &cancellables)
-
-        pinnedTabsViewModel.$dragMovesWindow.map(!)
-            .assign(to: \.pinnedTabsWindowDraggingView.isHidden, onWeaklyHeld: self)
-            .store(in: &cancellables)
-    }
```

Comment:
Same note as above, since `pinnedTabsViewModel` was always nil, it's safe to delete this one